posts/2020-12-16-rust-survey-2020.md

```diff
@@ -0,0 +1,118 @@
+
+When we asked to rate their expertise in Rust, there was a clear peak at 7 out of 10. It’s hard to say how this compares across languages but it seems notable that relatively few are willing to claim full expertise. However, when compared with last year, the Rust community does seem to be gaining expertise in the language. 
+
+![How would you rate your expertise in Rust?](/images/2020-12-rust-survey-2020/rust-expertise-overall.svg)
+
+We also asked about the difficulty of specific topics. The most difficult topic to learn according to survey results is somewhat unsurprisingly lifetime management with 61.4% of respondents saying that the use of lifetimes is either tricky or very difficult.  
+
+![Percent of respondents rating each topic as tricky or very difficult.](/images/2020-12-rust-survey-2020/topic-difficulty-ratings.svg)
+
+It does seem that having C++ knowledge helps with 20.2% of respondents with at least some C++ experience noting lifetimes to be “very difficult” while 22.2% of those without C++ knowledge found the topic to be “very difficult”. Overall, systems programming knowledge (defined as at least some experience in C and C++), tends to make for more confident Rust users: those with systems programming experience rating themselves as 5.5 out of 10 on their Rust expertise, while those with experience in statically typed garbage collected languages like Java or C# rating themselves as 4.9 out of 10. Those with only experience in dynamically typed languages like Ruby or JavaScript, rated themselves as 4.8 out of 10. 
+
+Unsurprisingly, the more often people use Rust, the more they feel they are experts in the language with 56.3% of those who use Rust daily ranking themselves as 7 or more out of 10 on how much of an expert they are on Rust compared with 22% of those who use Rust monthly. 
+
+![How would you rate your expertise in Rust? (Daily Rust users)](/images/2020-12-rust-survey-2020/rust-expertise-daily.svg)
+
+### Compile Times
+
+One continuing topic of importance to the Rust community and the Rust team is improving compile times. Progress has already been made with 50.5% of respondents saying they felt compile times have improved. This improvement was particularly pronounced with respondents with large codebases (10,000 lines of code or more) where 62.6% citing improvement and only 2.9% saying they have gotten worse. Improving compile times are likely to be the source of significant effort in 2021, so stay tuned!
+
+### Library Support 
+
+In general, respondents seemed please with the growing library support in the Rust ecosystem with 65.9% of respondents saying they had seen at least some improvement and only 4.9% saying they hadn't seen any improvement. When asked what type of library support was missing most, GUI programming was the overwhelming answer with only 26.9% of respondents noting that this was an area of improvement in the last year. 
```

Comment:
I think it should be `pleased` in `respondents seemed please`.

Reply:
Thanks for the eagle eyes!